MRN-Code/coinstac
# #MONOREPO

_Make it great. Make it MONO._

Asana task: https://app.asana.com/0/45796090191289/144732076040168
- **Problem:** constantly linking coinstac packages and dealing with mysterious cross-package bugs takes time away from development
- **Solution:** Use [lerna](https://lernajs.io) to place all coinstac packages in the same repository
- **Testing:**
  1. Pull this branch
  2. `npm i` in the root
  3. Run `npm run bootstrap`
  4. Run all packages tests and ensure :+1:
  5. Fire up server and UI to ensure they work
